src/main/resources/openapi.json

```diff
@@ -615,7 +615,7 @@
                         }
                     },
                     "404": {
-                        "description": "The specified topic was not found.",
+                        "description": "The path was not found.",
```

Comment:
I guess this overall 404 error was copied/pasted from the corresponding GET operation we already have but it's not still clear to me when this error should happen (there is also no test for it). I mean, you run a POST against an endpoint like `/topics/{topicname}` where the parameter on the path is a "input" parameter for the topic creation so it's clear it can't exist already (if you are creating it). When a NOT FOUND 404 should happen then?

Reply:
You're right, I copied that from another API. I removed it, if it is ok :)